packages/treasury/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "yarn build:bundles",
     "build:bundles": "node -r esm scripts/build-bundles.js",
-    "test": "ava --verbose",
+    "test": "ava",
```

Comment:
this is cool with @Chris-Hibbert and @katelynsills ?

Reply:
I tried running tests both ways, and didn't see a significant difference. I don't think I care either way. @michaelfig what difference is this supposed to make?

Reply:
It silences the test-by-test output.  The pattern I'd encourage is to use `yarn test -v` explicitly when you want that output.